ramls/request.json

```diff
@@ -282,6 +282,10 @@
     "itemLocationCode": {
       "description": "Allow specifying item location when creating title-level requests",
       "type": "string"
+    },
+    "isDcbReRequestCancellation": {
+      "description": "Indicates whether the request was cancelled during a DCB transaction update",
```

Comment:
The description needs more better wordings. It should reflect the real case for which this flag is added and will be utilized.
Please add test cases to this PR.